packages/curly/curly.0.2.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "Curly is a brain dead wrapper around the curl command line utility"
+maintainer: ["rudi.grinberg@gmail.com"]
+authors: ["Rudi Grinberg"]
+license: "ISC"
+homepage: "https://github.com/rgrinberg/curly"
+bug-reports: "https://github.com/rgrinberg/curly/issues"
+depends: [
+  "dune" {>= "2.4"}
+  "ocaml" {>= "4.02.3"}
+  "base-unix"
+  "result"
+  "alcotest" {with-test}
+  "cohttp-lwt-unix" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
    "@runtest" {with-test & ocaml:version >= "4.05.0"}
```

The issue, indeed, seem to be related to the tests. We could just disable them on 4.03